src/Controllers/SuperadminController.php

```
@@ -1066,6 +1078,85 @@ private function changeUserArchiveState(mixed $id, bool $active): void
+
+        $subscription = Subscription::findByOrgId($this->db, $orgId);
+        if ($subscription === null) {
+            FlashMessages::setLegacyError('Subscription not found.');
+            Response::applySecurityHeaders('app');
+            $this->response->redirect('/superadmin/organisations');
+            return;
+        }
+
+        $currentStatus = strtolower(trim((string) ($subscription['status'] ?? '')));
+        if ($pause && !in_array($currentStatus, ['active', 'trialing'], true)) {
+            FlashMessages::setLegacyError('Only active subscriptions can be paused.');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Core\FlashMessages' in method 'changeSubscriptionPauseState'. (undefined)

(StaticAccess)